server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json

```diff
@@ -292,60 +292,6 @@
             ],
             "title": "Population Below International Poverty Line in Rural Vs. Urban Areas"
           },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "sdg/SI_POV_EMP1.AGE--Y15T24",
-                      "sdg/SI_POV_EMP1.AGE--Y15T24__SEX--F",
-                      "sdg/SI_POV_EMP1.AGE--Y15T24__SEX--M"
-                    ],
-                    "title": "Employed Population 15 to 24 Years Old Below International Poverty Line, by Sex (%) in India",
```

Comment:
Wonder if this is from SDG refresh or some NL index change.

Reply:
hm, should not be from NL index change if there isn't any index changes in this PR?

Reply:
Ah, good point.  @n-h-diaz or @dwnoble - could you confirm please?

Reply:
There's still data in the refresh for these: https://autopush.datacommons.org/tools/statvar#sv=sdg/SI_POV_EMP1.AGE--Y15T24__SEX--F

Reply:
It doesn't look like there's any data in the refresh for Indian states for these variables

Reply:
so diffs are ok then?

Reply:
Yes looks like the data was removed with this refresh. I think it's OK to proceed, and ill bring this up with UN separately